recipes/gencove/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "2.4.3" %}
+{% set sha256 = "14d16e455142d798f680fa18b1d39377fbb0fea1d69f174bcb1b114038de5286" %}
+
+package:
+  name: gencove
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/gncv/gencove-cli/archive/refs/tags/{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  noarch: python
+  number: 0
+  script: '$PYTHON -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv '
+
+requirements:
+  build:
+    - pip
+    - setuptools
+  host:
+    - python >=3.7
+    - pytest-runner
+  run:
+    - python >=3.7
+    - python-dateutil >=2.2.0
+    - six >=1.5
+    - click >=7.0
+    - requests >=2.19.1
+    - boto3 >=1.17.97
+    - progressbar2 ==3.55.0
```

Comment:
Do these pins really need to be so precise? Is a range not possible?

Reply:
I'm not really sure, the requirements are copied from the repo, so I'm hesitant to change anything.
Also not my piece of software, so I've no idea

Reply:
Exact pins like this are usually a recipe for dependency hell in the future, but I won't fight you on it

Reply:
I wholly agree, but I also don't want to break stuff